### PR TITLE
🌐 Add Spanish traslation for `docs/tutorial/path-params-numeric-validations.md`

### PR DESCRIPTION
**Pull request by juanbeniteza**
Hi there guys

According to Spanish Translation feature: #1218 
This PR includes the Spanish translation for tutorial/path-params-numeric-validations.md

Thanks!

### CODE REVIEW

**Review by AdrianDeAnda — APPROVED** [review]

File: docs/es/docs/tutorial/path-params-numeric-validations.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+Digamos que tú quieres declarar el parámetro de query `q` como un `str` requerido.
+
+Y no necesitas declarar nada más para ese parámetro, es decir, realmente no necesitas usar `Query`.
+
+Pero si necesitas usar `Path` para el parámetro de path `item_id`.
+
+Python se quejará si colocas un atributo con un valor por defecto antes que un atributo que no tienes valor por defecto.
+
+Pero tu puedes reordenarlos, y tener el attributo sin valor por defecto (el parámetro de query `q`) primero.
```
Comment (israteneda) [inline docs/es/docs/tutorial/path-params-numeric-validations.md:41]:
```suggestion
Pero tu puedes reordenarlos, y tener el atributo sin valor por defecto (el parámetro de query `q`) primero.
```

File: docs/es/docs/tutorial/path-params-numeric-validations.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+Tú puedes declarar todos los mismos parámetros que para `Query`.
+
+Por ejemplo, para declarar un `title` como valor de metadata para el parámetro de path `item_id` puedes escribir:
+
+```Python hl_lines="10"
+{!../../../docs_src/path_params_numeric_validations/tutorial001.py!}
+```
+
+!!! nota
+    Un parámetro de path siempre es requerido ya que tiene que ser parte de la <abbr title="path">*ruta*</abbr>.
```
Comment (israteneda) [inline docs/es/docs/tutorial/path-params-numeric-validations.md:25]:
```suggestion
    Un parámetro de path siempre es requerido, ya que tiene que ser parte de la <abbr title="path">*ruta*</abbr>.
```

File: docs/es/docs/tutorial/path-params-numeric-validations.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+Y no necesitas declarar nada más para ese parámetro, es decir, realmente no necesitas usar `Query`.
+
+Pero si necesitas usar `Path` para el parámetro de path `item_id`.
+
+Python se quejará si colocas un atributo con un valor por defecto antes que un atributo que no tienes valor por defecto.
+
+Pero tu puedes reordenarlos, y tener el attributo sin valor por defecto (el parámetro de query `q`) primero.
+
+Para **FastAPI** esto no importa. FastAPI detectará los parámetros por sus nombres, tipos y declaraciones predeterminadas (`Query`, `Path`, etc), no se preocupa por el orden.
```
Comment (israteneda) [inline docs/es/docs/tutorial/path-params-numeric-validations.md:43]:
```suggestion
Para **FastAPI** esto no importa. FastAPI detectará los parámetros por sus nombres, tipos y declaraciones predeterminadas (`Query`, `Path`, etc.), no se preocupa por el orden.
```

File: docs/es/docs/tutorial/path-params-numeric-validations.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+Siendo así, puedes declarar tu función de la siguiente manera:
+
+```Python hl_lines="8"
+{!../../../docs_src/path_params_numeric_validations/tutorial002.py!}
+```
+
+## Ordena los parámetros como necesites, trucos
+
+Si quieres declarar el parámetro de query `q` sin un `Query` or cualquier valor por defecto, y el parámetro de path `item_id` usando `Path`, y tenerlos en un orden diferente, Python tiene una sintaxis especial para eso.
```
Comment (israteneda) [inline docs/es/docs/tutorial/path-params-numeric-validations.md:53]:
```suggestion
Si quieres declarar el parámetro de query `q` sin un `Query` o cualquier valor por defecto, y el parámetro de path `item_id` usando `Path`, y tenerlos en un orden diferente, Python tiene una sintaxis especial para eso.
```

File: docs/es/docs/tutorial/path-params-numeric-validations.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+
+Python no hará nada con ese `*`, pero sabrá que todos los parámetros siguientes deberán ser llamados como argumentos keyword (pares clave-valor), también conocidos como <abbr title="From: K-ey W-ord Arg-uments"><code>kwargs</code></abbr>. Incluso si no tienen un valor por defecto.
+
+```Python hl_lines="8"
+{!../../../docs_src/path_params_numeric_validations/tutorial003.py!}
+```
+
+## Validaciones numéricas: mayor o igual que
+
+Con `Query` y `Path` (y otros que verás más adelante) puedes declarar restriciones para strings, pero también restricciones para números.
```
Comment (israteneda) [inline docs/es/docs/tutorial/path-params-numeric-validations.md:65]:
```suggestion
Con `Query` y `Path` (y otros que verás más adelante) puedes declarar restricciones para strings, pero también restricciones para números.
```

File: docs/es/docs/tutorial/path-params-numeric-validations.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+    Todos ellos comparten los mismos parámetros de validaciones adicionales y metadata que ya has visto.
+
+!!! note "Detalles técnicos"
+    Cuando importas `Query`, `Path` y demás desde `fastapi`, ellos en realidad son funciones.
+
+    Que cuando son llamados, retornan instancias de clases del mismo nombre.
+
+    Entonces, tú importas `Query`, que es una función. Y cuando la llamas, la función retorna una instancia de la clase también llamada `Query`.
+
+    Estas funciones estan allí (en vez de usar las clases directamente) para que tu editor no marque errores por sus tipos.
```
Comment (israteneda) [inline docs/es/docs/tutorial/path-params-numeric-validations.md:121]:
```suggestion
    Estas funciones están allí (en vez de emplear las clases directamente) para que tu editor no marque errores por sus tipos.
```

File: docs/es/docs/tutorial/path-params-numeric-validations.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+
+    De esta manera puedes usar tu editor y herramientas para codificar sin tener que agregar configuraciones especiales para ignorar esos errores.
```
Comment (israteneda) [inline docs/es/docs/tutorial/path-params-numeric-validations.md:123]:
```suggestion
    De esta manera puedes utilizar tu editor y herramientas para codificar sin tener que agregar configuraciones especiales para ignorar esos errores.
```

File: docs/es/docs/tutorial/path-params-numeric-validations.md (hunk 1 above)
Comment (israteneda) [inline docs/es/docs/tutorial/path-params-numeric-validations.md:41]:
```suggestion
Pero tú puedes reordenarlos, y tener el attributo sin valor por defecto (el parámetro de query `q`) primero.
```

**Review by israteneda — CHANGES_REQUESTED** [review]
Buena traducción 🎉  solo algunos pequeños comentarios

**Comment by tiangolo** [comment]
As this PR had requested changes to be applied but has been inactive for a while, it's now going to be closed. But if there's anyone interested, feel free to create a new PR.

**Pull request closed without merging**